packages/ui-react/src/IdentityIcon/index.tsx

```diff
@@ -15,12 +15,15 @@ type Props = BaseProps & {
   value: string | Uint8Array
 };
 
-export default function IdentityIcon ({ className, size = 64, style, value }: Props) {
-  return (
-    <div
-      className={['ui--IdentityIcon', className].join(' ')}
-      ref={appendIcon(value, size)}
-      style={style}
-    />
-  );
+export default class IdentityIcon extends React.PureComponent<Props> {
+  render () {
+    const { className, size = 64, style, value } = this.props;
+    return (
+      <div
+        className={['ui--IdentityIcon', className].join(' ')}
+        ref={appendIcon(value, size)}
```

Comment:
I think this way of appending is probably a bit of an anti-pattern and/or wrong - it works, just not 100%. The append should probably be in this class and done in a more "React-y" way. (Out of scope for this PR maybe, but something to keep in mind)